joint_deconvolution/joint_deconvolution_and_feather_auto.py

```diff
@@ -0,0 +1,99 @@
+import json, os, sys, glob
+
+region      = sys.argv[1]
+pth         = os.path.abspath(os.getcwd())
+
+with open(f"./default_tclean_commands.json", "r") as fh:
+    tclean_commands = json.load(fh)
+
+os.chdir(pth+'/'+region)
+if not os.path.exists('clean_12m_7m'):
+    os.mkdir('clean_12m_7m')
+os.chdir(pth+'/'+region+'/clean_12m_7m/')
+
+vis_all     = glob.glob(pth+'/'+region+'/twelve_m/spws/*.spw3') + glob.glob(pth+'/'+region+'/seven_m/spws/*.spw3')
+if os.path.isdir(region+'_12m_7m_SPW3_concat.ms')==False:
```

Comment:
```suggestion
if not os.path.isdir(region+'_12m_7m_SPW3_concat.ms'):
```

`==False` is discouraged